core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly

```diff
@@ -27,29 +27,60 @@ THE SOFTWARE.
   <l:layout permission="${app.ADMINISTER}" title="${%Users}">
     <st:include page="sidepanel.jelly" />
     <l:main-panel>
-      <h1>${%Users}</h1>
+      <div class="jenkins-app-bar">
+        <div class="jenkins-app-bar__content">
+          <h1>
+            ${%Users}
+          </h1>
+        </div>
+      </div>
       <p>${%blurb}</p>
-      
-      <table class="sortable pane bigtable" id="people">
-        <tr>
-          <th class="minimum-width" data-sort-disable="true" />
-          <th>${%User ID}</th>
-          <th>${%Name}</th>
-          <th class="minimum-width" data-sort-disable="true" />
-        </tr>
-        <j:forEach var="user" items="${it.allUsers}">
+
+      <table class="jenkins-table sortable" id="people">
+        <thead>
           <tr>
-            <td><a href="${user.url}/" class="model-link inside"><img src="${h.getUserAvatar(user,'32x32')}" alt="" height="32" width="32" /></a></td>
-            <td><a href="${user.url}/">${user.id}</a></td>
-            <td><a href="${user.url}/">${user}</a></td>
-            <td>
-              <a href="${user.url}/configure"><l:icon class="icon-gear icon-md" /></a>
-              <j:if test="${user.canDelete()}">
-                <a href="${user.url}/delete"><l:icon class="icon-edit-delete icon-md" /></a>
-              </j:if>
-            </td>
+            <th class="jenkins-table__cell--tight" data-sort-disable="true" />
+            <th initialSortDir="down">${%User ID}</th>
+            <th>${%Name}</th>
+            <th class="jenkins-table__cell--tight" data-sort-disable="true" />
+            <th class="jenkins-table__cell--tight" data-sort-disable="true" />
           </tr>
-        </j:forEach>
+        </thead>
+        <tbody>
+          <j:forEach var="user" items="${it.allUsers}">
+            <tr>
+              <td>
+                <div class="jenkins-table__cell__button-wrapper">
+                  <a href="${user.url}/" class="jenkins-table__button model-link inside">
+                    <img src="${h.getUserAvatar(user,'32x32')}" alt="" height="32" width="32"/>
+                  </a>
+                </div>
+              </td>
+              <td>
+                <a href="${user.url}/" class="jenkins-table__link">${user.id}</a>
+              </td>
+              <td>
+                <a href="${user.url}/" class="jenkins-table__link">${user}</a>
```

Comment:
This was broken in https://github.com/jenkinsci/jenkins/pull/4803 but may as well fix it.

![image](https://user-images.githubusercontent.com/21194782/142587808-ace59d61-ff38-417e-be83-f277f0b6a6b5.png)

```suggestion
                <a href="${user.url}/" class="jenkins-table__link">${user.fullName}</a>
```